samples/topology/src/main/java/quarks/samples/topology/CombiningStreamsProcessingResults.java

```diff
@@ -63,6 +63,7 @@ public static void main(String[] args) throws Exception {
 
         // Split the stream by blood pressure category
         List<TStream<Map<String, Integer>>> categories = readings.split(6, new ToIntFunction<Map<String, Integer>>() {
+            private static final long serialVersionUID = 1L;
```

Comment:
@queeniema pls adjust this post merge if desired.   You should be able to replace the `new` with a lambda fn and simplify things.  e.g., something like
```
  ... readings.split(6, tuple -> {
                        if (tuple.get("Systolic") < 120 && ...
                            // Normal
                            return 0;
                        }  else if ...
                        ...
                    });
```